docs/content/about-iota/iota-architecture/transaction-lifecycle.mdx

```diff
@@ -17,127 +17,107 @@ import ThemedImage from '@theme/ThemedImage';
     }}
 />
 
-1. **Create a Transaction**: A user [submits](#submission) a transaction and signs it with a private key. The transaction can affect objects owned by the user, as well as [shared objects](../../developer/iota-101/objects/object-ownership/shared.mdx).
+1.  **Create and Sign a Transaction**: A user creates a transaction using a wallet application. This transaction can affect objects owned by the user, as well as [shared objects](../../developer/iota-101/objects/object-ownership/shared.mdx). The wallet application then signs the transaction with the user's private key.
 
-2. **Submit to Validators**: The IOTA protocol sends the transaction to every validator. The validators validate the transaction. If valid, the validators sign it and return the signed transaction to the client.
+2.  **Submit to Validators**: The client application communicates with a full node, which then distributes the transaction to all validators following the IOTA protocol. The validators verify the transaction. If valid, the validators sign it and return the signed transaction to the full node.
 
-3. **Form a Transaction Certificate**: After collecting responses from a supermajority of validators, the client can form a transaction _certificate_. Unlike consensus-based blockchains, IOTA validators are not burdened by needing to propagate signatures or aggregate _certificates_; this is the responsibility of the client or gateway.
+3.  **Form an Aggregate Certificate**: After collecting responses from a supermajority of validators, the full node forms an aggregate transaction _certificate_. Unlike consensus-based blockchains, IOTA validators are not burdened by needing to propagate signatures or aggregate _certificates_; this is the responsibility of the client or gateway.
```

Comment:
I think aggregate is used as a verb here. So you don't form an aggregate certificate. You aggregate certificates?

Anyway that I wanted to say is: I would keep calling it a Transaction Certificate here. It is the certificate you have after getting all the Validator responses, no?